src/modules/payer-svc/payer-svc.service.test.ts

```diff
@@ -352,4 +353,24 @@ describe('PayerService', () => {
       service.sendSmsVoucher('shortenHash', authUser),
     ).rejects.toThrow(new ForbiddenException(_403.ONLY_OWNER_CAN_SEND_VOUCHER));
   });
+
+  // it("should save information when we scan KYC face", async () => {
+  //   const authUser: JwtClaimsDataDto = {
+  //     sub: 'anotherPayerId',
+  //     type: UserRole.PAYER,
+  //     phoneNumber: mockPayer.user.phoneNumber,
+  //     names: `${mockPayer.firstName} ${mockPayer.lastName}`,
+  //     status: UserStatus.ACTIVE
+  //   };
+
+  //   const kycData: KYCDto = {
+  //     expire: "2017/05",
+  //     cardID: "ABCD",
+  //     birthday: "2002-05-10",
+  //     kyc: true
+  //   }
+  //   expect(async () =>
```

Comment:
you have to mock `payerRepository.findOne` and `payerRepository.save`

Reply:
But in this file `payerRepository.findOne` and `payerRepository.save` are already mock I think

Reply:
I see. So you should expect a success response and count the calls, like this
```
await service.updateKYC(authUser, kycData);
expect(payerRepository.save).toBeCalledTimes(1);
```